.github/ISSUE_TEMPLATE/funding_application.yml

```diff
@@ -0,0 +1,221 @@
+---
+name: TI Funding Request Form
+description: TI Funding Request Form OpenSSF
+title: "[Technical Initiative Funding Request]: "
+labels:
+  - administration
+  - For Review
+  - Funding Request
+body:
+  - type: markdown
+    attributes:
+      value: Thanks for taking the time to fill out this Funding Request.
+  - type: markdown
+    attributes:
+      value: |
+        **Pre-conditions:**
+        - The TI must have an identified lifecycle phase inside the OpenSSF, such as sandbox, incubating, or graduated. 
+        - The TI requesters should know 
+          - Within general funding limits for the TIs at the respective lifecycle, that are to be determined. 
+          - Within approved categories of asks (e.g. no asking for a TI outing at a fancy restaurant but yes to cloud credits)
```

Comment:
Is there a link we can provide TIs with a list of things (longer than just cloud credits) that could be considered valid requests?

Reply:
This came up in the TAC meeting that such a list should be made. Since I am not sure what the TAC will or should approve as a possible longer-scale solution, I have created a [GH project board](https://github.com/orgs/ossf/projects/25) That will build up over time with what the TAC has and hasn't granted funding for.